spec/models/document_type_selection/option_spec.rb

```diff
@@ -56,4 +56,23 @@
       expect(described_class.new(option).document_type?).to be true
     end
   end
+
+  describe "#pre_release?" do
+    it "returns true when the option is an existing document type and a pre-release feature" do
+      option = {
+        "id" => "policy_paper",
```

Comment:
I think we should do some stubbing here, as this test will start failing when we release publications, but I believe we want to retain the functionality for future document types.